ofl/notonaskharabic/METADATA.pb

```diff
@@ -0,0 +1,21 @@
+name: "Noto Naskh Arabic"
+designer: "UNKNOWN"
+license: "OFL"
+category: "SANS_SERIF"
```

Comment:
This and all Noto Serif fonts should be `category: "SERIF"`

Reply:
done